vita3k/gui/src/gui.cpp

```diff
@@ -329,6 +329,11 @@ void draw_ui(GuiState &gui, HostState &host) {
     if (gui.help_menu.about_dialog)
         draw_about_dialog(gui);
 
+    if (gui.live_area.live_area_dialog)
+        live_area_dialog(gui, host);
```

Comment:
```suggestion
        draw_live_area_dialog(gui, host);
```
for consistent naming